apiary.apib

```diff
@@ -407,6 +410,52 @@ We list all backwards-compatible additions here. These are currently available i
 - We now also return a `currency_exchange_rate` on all `.info` endpoints where we support multi-currency.  
   That is `quotations.info`, `invoices.info` and `creditNotes.info`.
 
+#### February 2019
+
+- We now also return the `invoicing_method` of a milestone. Both in `milestones.list` and `milestones.info`.
+
+- We now expose the invoiceability of time trackings and running timers. You can now use the `invoiceable` property on the following endpoints:
+  - `timeTracking.list`  
+  - `timeTracking.info`  
+  - `timeTracking.add`  
+  - `timeTracking.update`  
+  - `timers.current`  
+  - `timers.start`  
+  - `timers.update`  
+  We also return an `invoiceable` preference in the `users.me` that expose whether a user prefers to track time by default as invoiceable or not.
+
+- We've added a `national_identification_number` property on contacts.
```

Comment:
This has also been added to companies, we should probably add that in here?

Reply:
Wasn't it already available there as `local_business_number` and isn't it in the list of BC breaking changes?

Reply:
Ah, that's right